src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,68 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { CloudExplorerV1 } from 'vscode-kubernetes-tools-api';
+
+const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
+const configurePipelineCommand = 'configure-cicd-pipeline';
```

Comment:
💡 Idea: Either use this const in all the places where you are passing commands, or I believe you already have a mechanism to pass through chain of functions so you can remove this and command declaration can reside inside the function?